etna/lib/etna/clients/janus/client.rb

```diff
@@ -10,8 +10,7 @@ class Janus < Etna::Clients::BaseClient
       def get_project(get_project_request = GetProjectRequest.new)
         json = nil
         @etna_client.get(
-          "/project/#{get_project_request.project_name}",
-          get_project_request) do |res|
+          "/api/admin/#{get_project_request.project_name}/info") do |res|
```

Comment:
If you are editing `etna` gem, should you also bump the version in the `etna.gemspec` file?